docs/esa/jwst/jwst.rst

```diff
@@ -225,68 +224,68 @@ To query the data products associated with a certain Observation ID
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> product_list=Jwst.get_product_list(observation_id='jw00777011001_02104_00001_nrcblong')
-  >>> for row in product_list:
-  >>>     print("filename: %s" % (row['filename']))
+  >>> product_list = Jwst.get_product_list(observation_id='jw01063107001_02101_00013_nrca3')
+  >>> print(product_list['filename'])
+
+                        filename
+    ------------------------------------------------
+    jw01063-o107_20220328t013707_image2_144_asn.json
+            jw01063107001_02101_00013_nrca3_cal.fits
+             jw01063107001_02101_00013_nrca3_cal.jpg
+       jw01063107001_02101_00013_nrca3_cal_thumb.jpg
+            jw01063107001_02101_00013_nrca3_i2d.fits
+    ...
 
-  filename: jw00777011001_02104_00001_nrcblong_c1005_crf.fits
-  filename: jw00777011001_02104_00001_nrcblong_cal.fits
-  filename: jw00777011001_02104_00001_nrcblong_cal.jpg
-  filename: jw00777011001_02104_00001_nrcblong_cal_thumb.jpg
-  filename: jw00777011001_02104_00001_nrcblong_i2d.fits
-  filename: jw00777011001_02104_00001_nrcblong_o011_crf.fits
 
 You can filter by product type and calibration level (using a numerical value or the option 'ALL' -set by default- that will download
 all the products associated to this observation_id with the same and lower levels).
 
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> product_list=Jwst.get_product_list(observation_id='jw97012001001_02101_00001_guider1', product_type='science')
-  >>> for row in product_list:
-  >>>     print("filename: %s" % (row['filename']))
+  >>> product_list = Jwst.get_product_list(observation_id='jw01023029001_02101_00004_mirimage', product_type='science')
+  >>> print(product_list['filename'])
+
+                         filename
+    -------------------------------------------------
+    jw01023029001_02101_00004_mirimage_c1006_crf.fits
+          jw01023029001_02101_00004_mirimage_cal.fits
+          jw01023029001_02101_00004_mirimage_i2d.fits
+     jw01023029001_02101_00004_mirimage_o029_crf.fits
+         jw01023029001_02101_00004_mirimage_rate.fits
+    ...
 
-  filename: jw97012001001_02101_00001_guider1_cal.fits
-  filename: jw97012001001_02101_00001_guider1_uncal.fits
 
 To download a data product
 
 .. code-block:: python
 
   >>> from astroquery.esa.jwst import Jwst
-  >>> query="select a.artifactid, a.uri from jwst.artifact a, jwst.plane p where p.planeid=a.planeid and p.obsid='00000000-0000-0000-9c08-f5be8f3df805'"
-  >>> job=Jwst.launch_job(query, async_job=True)
-  >>> results=job.get_results()
+  >>> query = "select o.observationid, a.artifactid, a.filename from jwst.observation o join jwst.artifact a on a.obsid = o.obsid where o.proposal_id = '01166' and o.intent = 'science'"
```

Comment:
maybe for `o` and `a` also use something a bit longer (obs and art? though those are not that much better either). Anyway, take this as my thinking out loud, I don't in fact have a strong preference just wonder what would work nicer.

Also, it is worth looking at the rendered docs and breaking long lines rather than having one very long in the horizontal  scroll

Reply:
Fixed